example/tests/test_models.py

```diff
@@ -60,16 +72,20 @@ def test_template_should_be_checked(self):
             self.render('title', {'article': DeletedRelatedObject()})
 
         # check multiple formats of single variable
-        for template in ('{{article}}', '{{ article }}', '{{ article|safe }}'):
+        for template in ('Article: {{article}}', 'Article: {{ article }}', 'Article: {{ article|safe }}'):
```

Comment:
Better to render static text with variable, than variable only. (there was a bug regarding this in the previsous version of this PR)